tests/factories/user_category_factories.py

```diff
@@ -0,0 +1,12 @@
+from tests.factories import *
+
+from sourcemash.models import UserCategory, Category
+from user_factories import UserFactory
+
+class UserCategoryFactory(SQLAlchemyModelFactory):
+    FACTORY_FOR = UserCategory
+    FACTORY_SESSION = db.session
+
+    user = factory.SubFactory(UserFactory)
+    category = Category("category")
```

Comment:
What's weird about categories made this way in the factory is that they won't actually be present in the database i don't think. So if you wanted to access category in a test, you'll get an error saying it's not bound to the session or something like that